### PR TITLE
EDU-709 - /cloud/service-availability updates for GCP

### PR DESCRIPTION
**Pull request by MasonEgger**
Adds regions for GCP

### CODE REVIEW

File: docs/evaluate/temporal-cloud/service-availability.mdx [hunk 1]
```diff
@@ -51,6 +53,18 @@ Currently, Temporal Cloud operates in several regions on Amazon Web Services (AW
 
 _**Your Workers and Client code aren't required to be hosted on AWS.**_
 
+### GCP
+
+Currently, Temporal Cloud operates in two regions on Google Cloud (GCP):
```
Comment (jsundai) [inline docs/evaluate/temporal-cloud/service-availability.mdx:58]: You can remove the word currently.

**Review by jsundai — APPROVED** [review]
Remove the word currently and it's good to go! Great work.

File: docs/evaluate/temporal-cloud/service-availability.mdx (hunk 1 above)
Comment (MasonEgger) [reply docs/evaluate/temporal-cloud/service-availability.mdx]: Fixed!

File: docs/evaluate/temporal-cloud/service-availability.mdx [hunk 2]
```diff
@@ -51,6 +53,18 @@ Currently, Temporal Cloud operates in several regions on Amazon Web Services (AW
...
 _**Your Workers and Client code aren't required to be hosted on AWS.**_
 
+### GCP
+
+Temporal Cloud operates in two regions on Google Cloud (GCP):
+
+| Area          | Code                 | Region            |
+| ------------- | -------------------- | ----------------- |
+| North America | us-west1             | Oregon            |
+| Asia Pacific  | australia-southeast1 | Sydney            |
```
Comment (XaF) [inline docs/evaluate/temporal-cloud/service-availability.mdx:63]: I'm very confused as to why for some we put the state, and for others we put the city 🤷‍♂️

File: docs/evaluate/temporal-cloud/service-availability.mdx [hunk 3]
```diff
@@ -51,6 +53,18 @@ Currently, Temporal Cloud operates in several regions on Amazon Web Services (AW
...
+
+
+_**Your Workers and Client code aren't required to be hosted on GCP.**_
```
Comment (XaF) [inline docs/evaluate/temporal-cloud/service-availability.mdx:66]: Can this statement be put in a more generic way in the page?
e.g. from the sentence just above the split between AWS and GCP, "here are the regions that Temporal Cloud can provide a server in, but your Workers and Client code is not required to be hosted in the same region, of the same cloud provider. We however advise on creating your Namespace in a region geographically close to your Workers' hosting location."

File: docs/evaluate/temporal-cloud/service-availability.mdx (hunk 2 above)
Comment (fairlydurable) [reply docs/evaluate/temporal-cloud/service-availability.mdx:63]: I hope this helps: 
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

Comment (XaF) [reply docs/evaluate/temporal-cloud/service-availability.mdx:63]: Sorry, I said "we" but I know a lot of providers do it. I think on Google documentation they do put the city though, but always feels weird we don't use the same unit of precision!

File: docs/evaluate/temporal-cloud/service-availability.mdx (hunk 3 above)
Comment (MasonEgger) [reply docs/evaluate/temporal-cloud/service-availability.mdx:66]: Should we just remove this? We also have it for AWS. The above paragraph covers the point, so we don't necessarily need this in this location.

Comment (XaF) [reply docs/evaluate/temporal-cloud/service-availability.mdx:66]: Works for me

**Comment by MasonEgger** [comment]
Good for Replay release

File: docs/evaluate/temporal-cloud/service-availability.mdx (hunk 3 above)
Comment (XaF) [inline docs/evaluate/temporal-cloud/service-availability.mdx:66]: Forgot to remove that for both clouds?

**Review by XaF — APPROVED** [review]

File: docs/evaluate/temporal-cloud/service-availability.mdx (hunk 3 above)
Comment (MasonEgger) [reply docs/evaluate/temporal-cloud/service-availability.mdx]: Removed.

**Pull request merged**